PlexRequests.UI/Views/Shared/Partial/_Navbar.cshtml

```diff
@@ -38,10 +39,11 @@
                 {
                     <li><a id="donate" href="https://www.paypal.me/PlexRequestsNet" target="_blank"><i class="fa fa-heart" style="color: red"></i> @UI.Layout_Donate</a></li>
                 }
+                <li id="customDonate" style="display: none"><a id="customDonateHref" href="https://www.paypal.me/PlexRequestsNet" target="_blank"><i class="fa fa-heart" style="color: yellow;"></i> <span id="donationText">Donate to Library Maintainer</span></a></li>
```

Comment:
I added it to the resources. Seems like if it doesn't exist in translations it just defaults to English? I that just how it is for now, and at some point someone translates it, or should I just run it through google translate? I'm not sure how localization works for open source stuff, for work we get it done in batches of thousands of strings at a time, but sometimes I'm not sure how much better than google it is.